src/lib/utils/events.ts

```diff
@@ -0,0 +1,78 @@
+type EventArgs = {
+  event:
+    | 'createCartError'
+    | 'addLineItemError'
+    | 'updateLineItemError'
+    | 'removeLineItemError'
+    | 'createCartSuccess'
+    | 'addLineItemSuccess'
+    | 'updateLineItemSuccess'
+    | 'removeLineItemSuccess'
+  callback: (...args: any[]) => any
+}
+type Events = { [k: string]: EventArgs['callback'][] }
+
+export const createEventManager = () => {
```

Comment:
Why don't we use the "EventEmiter" stdlib here? Is there any advantage of creating our own handlers?

Reply:
Using stdlib on last commit ✅
cc @matiasperz @julianbenegas